packages/gatsby-plugin-preact/src/__tests__/gatsby-node.js

```diff
@@ -0,0 +1,11 @@
+const { onCreateWebpackConfig } = require(`../gatsby-node`)
+
+describe(`gatsby-plugin-preact`, () => {
+  it(`sets the correct webpack config`, () => {
```

Comment:
Let's split these out.

```js
it(`invokes setWebpackConfig with correct config when not develop-html`, () => {
  const setWebpackConfig = jest.fn()
  onCreateWebpackConfig({ stage: `develop`, actions: { setWebpackConfig } })

  expect(setWebpackConfig).toHaveBeenCalledTimes(1)
  // I like to be more explicit here, but up to you!
  expect(setWebpackConfig).toHaveBeenCalledWith({
    resolve: {
      alias: {
        create-react-class: '// etc',
      }
    }
  })
})

it(`does not invoke setWebpackConfig when stage is develop-html`, () => {
  const setWebpackConfig = jest.fn()
  onCreateWebpackConfig({ stage: `develop-html`, actions: { setWebpackConfig } })

  expect(setWebpackConfig).not.toHaveBeenCalled()
})
```

Reply:
Thanks for having a look, I've now split these out  🙂